src/InputWithLabel.js

```diff
@@ -0,0 +1,24 @@
+import React,{useEffect,useRef} from "react";
+
+
+function InputWithLabel(props) { 
```

Comment:
destructuring the props would help clean up your input in the return statement so you don't have to keep repeating props.x and props.y